hyper_click_annotator.py

```diff
@@ -71,9 +71,8 @@ def annotate(self, point):
             matched = self.is_valid_line(line_content)
 
             if matched:
-                destination_str = matched.group(1)
-                file_path = HyperClickPathResolver(
-                    destination_str, v.file_name(),
+                file_path = HyperClickPathResolver(v,
+                    matched, v.file_name(),
```

Comment:
since you pass `v`, you don't have to pass `v.file_name()` too

Reply:
i will make the change, i haven't notice that :+1: